FeedAPIChallenge/Feed API/FeedItemsMapper.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FeedItemsMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Chris on 2021-05-02.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedItemsMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+		var feed: [FeedImage] {
+			return items.map { $0.item }
+		}
+	}
```

Comment:
`internal` is a default access level. It is ok to explicitly declare it. On the other hand it is more important to be consistent across the codebase. Therefore if some class or method is explicitly marked as `internal` then all other should be marked as well. For example `Root.items` should be marked as well `Root.feed`. This project doesn't use explicit `internals` so it should be either added everywhere or removed here.

Please remove `internal` access modifier to follow the code convention of the codebase.
```suggestion
final class FeedItemsMapper {
	private struct Root: Decodable {
		let items: [Item]
		var feed: [FeedImage] {
			return items.map { $0.item }
		}
	}
```

Reply:
Good eye!